.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -131,8 +131,10 @@ jobs:
       env:
         STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
         STACKROX_NVD_API_CALL_INTERVAL: 6s
-        STACKROX_NVD_ZIP_PATH: nvd.zip
```

Comment:
This isn't the only file we use this env var with this same setting. Is this not a problem for the dev updater, for example?

Reply:
I checked the dev file, it doesn't change dirs, so think its OK

Reply:
Double-checked [latest `dev` run](https://github.com/stackrox/stackrox/actions/runs/10425893052/job/28877886962) and it appears OK (this log wouldn't appear if was an error):

**Run updater (single bundle):**

```
{"level":"info","component":"enricher/nvd/Enricher/Configure","bundle":"nvd","feed_path":"nvd.zip","time":"2024-08-16T20:39:03Z","message":"enricher configured with feed path"}
```

**Run updater (multi bundle):**

```
{"level":"info","component":"enricher/nvd/Enricher/Configure","bundle":"nvd","feed_path":"nvd.zip","time":"2024-08-16T20:39:03Z","message":"enricher configured with feed path"}
```